scripts/register_service_account/README.md

```diff
@@ -10,3 +10,27 @@ Usage (from the main directory where run.sh resides):
 Usage (using Docker):
 
 ```docker run --rm -it -v "$HOME"/.config:/.config -v <path to your service account credentials json file>:/svc.json broadinstitute/firecloud-tools python /scripts/register_service_account/register_service_account.py -j /svc.json -e <email address for owner of this service account, it's where notifications will go>```
+
+## Register a service account for use in FireCloud without a JSON credentials keyfile
+Some organizations don't allow downloads of json keyfiles. In this case, it is still possible to run a VM with the credentials of a service account, and then use those credentials to make the request. You will need to run the modified version of the script in order to do this.
+
+1. Create a VM with the service account's identity
+    1. Navigate to the Create new Instance page in GCP for the project containing the service account
+    2. Choose anything for name, size, and region. 
+    3. Most importantly **be sure that the service account** is selected in the Identity and API access section
+    4. We also need to enable the email, profile, and openid scopes to the service account. Unfortunately the UI doesn't allow us to add scopes for these kinds of service accounts. However if we create the VM through a gcloud command, we can add the scopes. **Do not click create**, instead click on "Equivalent Command Line" and copy the command.    
+    5.  Edit the command to add the email, profile and openid scope. Run this command. For example, my command looks like:
+```gcloud compute instances create instance-1 --project=terra-foo-foo-dev --zone=us-central1-a --machine-type=e2-medium --network-interface=network-tier=PREMIUM,subnet=default --maintenance-policy=MIGRATE --provisioning-model=STANDARD --service-account=foo-example-sa@terra-foo-foo-dev.iam.gserviceaccount.com --scopes=https://www.googleapis.com/auth/cloud-platform,email,profile,openid --create-disk=auto-delete=yes,boot=yes,device-name=instance-1,image=projects/debian-cloud/global/images/debian-11-bullseye-v20220920,mode=rw,size=10,type=projects/terra-foo-foo-dev/zones/us-central1-a/diskTypes/pd-balanced --no-shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --reservation-affinity=any```
```

Comment:
Consider making this a bit more readable with newlines and '\\' continuation characters.